pkg/httputil/logging.go

```diff
@@ -54,6 +56,30 @@ func RequestID(r *http.Request) (*http.Request, string) {
 	return r, reqID
 }
 
+func ClientActionName(r *http.Request) string {
+	ctx := r.Context()
+	resp := ctx.Value(ClientActionNameKey)
+	UIReqID := ""
+	if resp != nil {
+		UIReqID = resp.(string)
+	}
+	return UIReqID
+}
+
+func SessionID(r *http.Request) (*http.Request, string) {
+	ctx := r.Context()
+	resp := ctx.Value(SessionIDContextKey)
```

Comment:
`resp` makes me think of "response", I am not sure that name is relevant here.  Maybe
```suggestion
	sessionID := ctx.Value(SessionIDContextKey)
```
?